setup.py

```diff
@@ -82,11 +82,17 @@ def get_petsc_dir():
       author="Imperial College London and others",
       author_email="firedrake@imperial.ac.uk",
       url="http://firedrakeproject.org",
-      packages=["firedrake", "firedrake.mg", "firedrake.slope_limiter",
-                "firedrake.matrix_free", "firedrake.preconditioners",
+      packages=["firedrake",
+                "firedrake.adjoint",
                 "firedrake.cython",
+                "firedrake.mg",
+                "firedrake.slope_limiter",
+                "firedrake.matrix_free",
+                "firedrake.preconditioners",
                 "firedrake.slate", "firedrake.slate.slac", "firedrake.slate.static_condensation",
-                "firedrake_configuration", "firedrake_citations"],
+                "firedrake_configuration",
+                "firedrake_citations"
+                ],
```

Comment:
Thanks. Perhaps we should instead use `setuptools.find_packages()` instead, to make this future proof. WDYT?

Reply:
Done, retested, and force-pushed.

Reply:
Great, thanks!